core/src/main/java/org/apache/calcite/rel/rules/ProjectJoinTransposeRule.java

```diff
@@ -157,7 +159,24 @@ public ProjectJoinTransposeRule(
   @Value.Immutable(singleton = false)
   public interface Config extends RelRule.Config {
     Config DEFAULT = ImmutableProjectJoinTransposeRule.Config.builder()
-        .withPreserveExprCondition(expr -> !(expr instanceof RexOver))
+        .withPreserveExprCondition(expr -> {
+          // Non push down over's expression by default
```

Comment:
```suggestion
          // Do not push down over's expression by default
```

Reply:
kindly ping.

Reply:
@wojustme Here is one missing comment which hasn't been addressed.

Reply:
@libenchao Thanks for your reminder, but I check this pr, and I found that I have update this code, screenshot as follows:
![image](https://user-images.githubusercontent.com/8967656/198003453-016b8697-b580-4c8f-8448-46afcd2d10be.png)

Reply:
It's in different line, I'm saying line 163

Reply:
I'm sorry for it.
Please review it, thanks a lot.
@libenchao